bibliopixel/drivers/serial/devices.py

```diff
@@ -33,7 +33,7 @@ def find_serial_devices(self):
             port = ports[0]
             id = self.get_device_id(port, self.baudrate)
             ver = self._get_device_version(port, self.baudrate)
-            if len(ports) > 1:
+            if isinstance(ports, list) and len(ports) > 1:
```

Comment:
maybe this was fine on Mac but on Linux/Windows `ports` is not always a list (or it was never tested on Mac either).